lib/api/objectDelete.js

```diff
@@ -140,11 +140,9 @@ export default function objectDelete(authInfo, request, log, cb) {
                 resHeaders['x-amz-version-id'] = result.versionId === 'null' ?
                     result.versionId : versionIdUtils.encode(result.versionId);
             }
-            // TODO metric should be 'deleteObject', not 'putObject',
-            // but number of objects should be incremented since putting a new
-            // version. Utapi will be updated to allow this
-            pushMetric('putObject', log, { authInfo, bucket: bucketName,
-                newByteLength: 0, oldByteLength: 0 });
+            // We increment deleteObject even if creating a delete marker
```

Comment:
I could add a clarifying comment about this. The method called in utapi 'putDeleteMarkerObject' actually increments the 's3:deleteObject' metric. The intention of adding this method was to encapsulate the logic of putting a delete marker (i.e., incrementing both 'deleteObject' and 'numberOfObjects' at once).